com.moura.Progress.json

```diff
@@ -0,0 +1,153 @@
+{
+    "id": "com.moura.Progress",
```

Comment:
is this software affiliated with the Brazilian company ? Or did you just pick a random domain?

Reply:
Honestly, I had picked a random domain. Is this domain already being used?

Reply:
Would it appropriate if I update the domain to something connected to my github account? Because if that's the case, I'll gladly fix it along to the other required changes

Reply:
Yet you acknowledged the following:

> The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

Reply:
Indeed. I'm working on fixing the remaining issues